content/en/docs/Responses/ReplayGain.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "ReplayGain"
+linkTitle: "ReplayGain [OS]"
+opensubsonic:
+- Addition
+description: >
+  The replay gain data of a song.
+---
+
+{{< tabpane persistLang=false >}}
+{{< tab header="**Example**:" disabled=true />}}
+{{< tab header="OpenSubsonic" lang="json">}}
+{
+    "trackGain": 0.1,
+    "albumGain": 1.1,
+    "trackPeak": 9.2,
+    "albumPeak": 9,
+    "baseGain": 0
+}
+{{< /tab >}}
+{{< tab header="Subsonic"  >}}
+Does not exist.
+{{< /tab >}}
+{{< /tabpane >}}
+
+| Field |  Type | Req. | OpenS. | Details |
+| --- | --- | --- | --- | --- |
+| `trackGain` | `number` | No | **Yes**    | The track replay gain value. (In Db) |
+| `albumGain` | `number` | No | **Yes**    | The album replay gain value. (In Db) |
+| `trackPeak` | `number` | No | **Yes**    | The track peak value. (Range [0.0-1.0]) |
+| `albumPeak` | `number` | No | **Yes**    | The album peak value. (Range [0.0-1.0]) |
```

Comment:
Small comment here, but the example fields `trackPeak`, `albumPeak` do not match the "Details" section, which limits it to a range of [0.0, 1.0]. The peak can [definitely exceed 1.0 though](https://wiki.hydrogenaud.io/index.php?title=ReplayGain_specification#Peak_amplitude), so I'd recommend updating the details section

Reply:
@kgarner7 I must admit I only use the Gain values myself they are based on the comment from https://github.com/opensubsonic/open-subsonic-api/discussions/10 @dweymouth but yes from that doc it seems it can go higher even if not seen from Plex data for example.

Reply:
Makes sense! I do have some files of my own that are tagged with peak > 1.0, so I wanted to chime in here

Reply:
Yeah @kgarner7 is right - replayGain peaks of a bit over 1 are definitely possible. It was an oversight on my part in #10 to specify a [0..1] range.